### ISSUE TITLE
Can WCAG add caption standards, so that captions are readable, usable and benefit the people they are intended to assist?

### ISSUE THREAD

**Issue by JLS2024** [w3c/wcag3#119]
WCAG recommends captions, but fails to include standards. Captions need standards (https://janicelintz.com/wp-content/uploads/2015/05/multichannelviewpt-72913.pdf) to be readable, usable, and benefit the people they are intended to assist. Can w3c/wcag3 add captioning standards? 

I recommend the standards that the Association of National Advertisers recommends(https://janicelintz.com/wp-content/uploads/2015/03/ana_closedcaption_whitepaper-f.pdf) These standards are the basis for the FCC standards (https://www.fcc.gov/sites/default/files/closed_captioning_on_television.pdf) and the NPS captioning standards (https://janicelintz.com/wp-content/uploads/2020/08/19-10-npsguidelines.pdf). 

Janice S. Lintz
Hearing Loss Changemaker

2023 [Presidential Management Fellow](https://apply.pmf.gov/finalists.aspx) Finalist
2023 Forbes 50/50 Finalist
2022 Congressional Recognition by Congressmember Paul Tonko
2022 [NYS Disability Rights Hall of Fame Inductee](https://nysilc.org/hof)
2021 Cited in the Proposed [FDA OTC Hearing Aid Regulations](https://public-inspection.federalregister.gov/2021-22473.pdf)’ footnotes
2018 Bill and Melinda Goalkeepers Foundations Partnership with TPG Recipient
2016 Aspen Institute Spotlight Health Scholar
2016 Nominated United State of Women Changemaker
2016 Acknowledged in the [National Park Service Accessibility Guidelines](https://janiceslintz.files.wordpress.com/2020/08/19-10-npsguidelines.pdf) (P76)
2008 [People Magazine Hero](https://janiceslintz.files.wordpress.com/2014/11/people.jpg)

**Comment by shawna-slh** [comment]
Perhaps appropriate resources can be added to [Understanding [SC 1.2.2] Captions, Related Resources](https://www.w3.org/WAI/WCAG22/Understanding/captions-prerecorded.html#resources)

**Comment by JLS2024** [comment]
Hi, Understanding captions is not the same as caption standards. Standards are placement, size, and what should be included. Understanding is why captions are critical. That is a big difference.

Best,

Janice

**Comment by shawna-slh** [comment]
The [**Related Resources** section of Understanding [SC 1.2.2] Captions](https://www.w3.org/WAI/WCAG22/Understanding/captions-prerecorded.html#resources) is the place to put links to such standards.

**Comment by JLS2024** [comment]
There is a difference between here is something to learn about and here is what you must do. Everything in WCAG is required by the federal government or Access Board. Caption standards should be a must and not a should. 

Please provide me a link to the captions working group since I could not locate it. Thanks!

**Comment by patrickhlauke** [comment]
WCAG is used/referenced beyond the US. It would be mighty strange to enshrine a specific standard from one country to be mandated worldwide.

**Comment by JLS2024** [comment]
The standard is what people who are deaf and hard of hearing need. It does not need to reference the specific ANA standard but what is contained in the standard. Access either works or it doesn’t. The font size, the location and information does not change by country.

**Comment by JLS2024** [comment]
Please also provide me the link to the captioning working group since I cannot find it. Can I please also have a list of all members.

**Comment by nschonni** [comment]
https://www.w3.org/TR/webvtt1/ is probably the spec you're talking about. To find a group, they're all listed on https://www.w3.org/groups/wg/

**Comment by JLS2024** [comment]
This wouldn’t be the correct group. I do not believe there is any group addressing captioning standards, which is the problem. Captions alone without standards are pointless. It’s why the FCC created standards. How can a group be started or another group tackle creating standards so the caption placement does not cover up content, words are spelled correctly, the font size is readable, non-word content is incorporated and so on?

**Comment by nschonni** [comment]
> How can a group be started or another group tackle creating standards so the caption placement

All the information about how groups are formed, and what they address is in https://www.w3.org/policies/process/

**Comment by JLS2024** [comment]
I feel like I am being asked to jump through one hoop after another. Fill out one form, join this site etc to avoid solving the issue. Please stop giving me the work and please figure out how to implement a solvable problem. Sent from my iPhoneOn Oct 6, 2024, at 3:04 PM, Nick Schonning ***@***.***> wrote:﻿

How can a group be started or another group tackle creating standards so the caption placement

All the information about how groups are formed, and what they address is in https://www.w3.org/policies/process/

**Comment by detlevhfischer** [comment]
@JLS2024 I understand your frustration. Since it is unlikely that a WCAG version 2.3 will be realeased, efforts now focussing on WCAG 3, this may be the best place to submit proposals for inclusion of captions characteristics https://www.w3.org/TR/wcag-3.0/#guidelines In there, you find 
> [Caption control (Exploratory)](https://www.w3.org/TR/wcag-3.0/#x2-12-5-caption-control): The position and formatting of captions can be changed.

For countries adopting the European Standard [EN 301 549 (PDF)](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), there is already the requirement 7.1.4 Captions characteristics that lists a number of parameters that users should be able to change (not all of them offered in the different environments and players today).

**Comment by JLS2024** [comment]
It seems there may be a misunderstanding regarding the scope and focus of the working group. From the communications we've exchanged, as well as the number of groups and posts you’ve directed me to, it appears there is no one in the working group who has hearing loss, or else this issue would have been addressed earlier.

I have asked multiple times for clarification on which group is addressing hearing access and who is working on this issue. Instead, I’ve been sent down several paths, only to find that no specific group is focused on this, and no one is actively working on it. I should not be required to file additional paperwork for an issue that is already well-documented and obvious.

I have provided the necessary documentation, and now I need to know how this can be implemented. At this point, I am returning the responsibility of this work to the group. With over 22 years of experience in hearing access, including significant global projects, I can recognize when efforts are being diverted, and it appears this is what's happening here.

Please let me know how the group plans to proceed.

**Comment by mraccess77** [comment]
In the US The FCC also has some requirements worth considering for the display of captions.
§ 79.103 Closed caption decoder and display requirements for apparatus.
https://www.law.cornell.edu/cfr/text/47/79.103

**Comment by detlevhfischer** [comment]
> Please let me know how the group plans to proceed.

@JLS2024 You are receiving individual answers here, from people who cannot speak for the group as a whole. Maybe it would be better to directly contact the chairs of AGWG <group-ag-chairs@w3.org> and ask them if a task force focused on hearing loss can be set up.

**Comment by iadawn** [comment]
@JLS2024 Thanks for raising the issue. The [Accessibility Guidelines Working Group (AGWG)](https://www.w3.org/groups/wg/ag/) is responsible for the development of [WCAG 3](https://www.w3.org/WAI/standards-guidelines/wcag/wcag3-intro/) and the maintenance of [WCAG 2](https://www.w3.org/WAI/standards-guidelines/wcag/). 

* [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) provides a good introduction to accessibility and the breadth of the work at W3C on the topic.
* [W3C Accessibility Standards Overview](https://www.w3.org/WAI/standards-guidelines/) provides more of an introduction to the standards themselves.
 
WCAG 3 is taking a slightly different approach to WCAG 2 with more of an emphasis on Outcomes and what can be done to best deliver against those outcomes to improve access to digital content for people with disabilities.

We are currently developing content for each of the Outcomes. This is conducted in sub-groups of the working group. We have not as yet started a sub-group on captions but it will certainly happen and the issue you have raise will be part of the considerations.

I will respond in a separate email with information on how to become more involved if that is something you are interested in doing.

Thanks

Kevin
AGWG Team Contact

**Comment by bruce-usab** [comment]
Welcome Janice!
As you know, the FCC caption quality standards are tuned to U.S. broadcast television, but are readily applicable to about any video media.  They are performance standards, and work well enough IMHO, but I am not convinced they can reasonably be converted into a mathematical algorithm.

**Comment by JLS2024** [comment]
Hi, Bruce-

It's great to hear from you again. I’ll admit, algorithms aren’t my area of expertise—getting through Harvard’s quant class was no easy feat. But when it comes to hearing access, that’s where my passion lies. Where there's a will, there's a way. If AI can handle complex tasks, I believe there's an algorithm for everything.

For years, I've faced obstacles to hearing access—many of which seemed deliberately placed in my path. You and I discussed this on other issues. Yet, I’ve never been deterred. 

Take the Smithsonian, for instance—they gave me every excuse under the sun for why they couldn’t implement access. But after 22 years of persistence, the National Air & Space Museum now has induction loops. This just goes to show: no matter the resistance, When there’s a will, there’s always a way. 

￼


Best,

Janice

**Comment by JLS2024** [comment]
Kevin-

Wonderful. I would like to be part of the group.

Best,

Janice

**Comment by alastc** [comment]
Looking ahead to when we work on the multimedia guidelines, I think it is very unlikely that we (the working group) would develop a new standard for captions, particularly when that work has already been done by other organisations.
 
A couple of other reasons include:
- The (reliable) testability of a captioning standard is unlikely to be in line with the “requirement” level in WCAG 3.
- One captioning standard may not work across the whole world. For example, some languages may not have a concept of capitalisation, and the best location on screen might vary between regions.
 
However, what we could do is include an assertion along the lines of: “We (the organisation making a claim) have followed the appropriate regional standard for captions”. That assertion could be required to reach a certain level of conformance with WCAG. So there would be a requirement for captions, then an assertion to tackle the quality of captions.

Our informative documents could also link to known (good) captioning standards from around the world.

**Comment by alastc** [comment]
A few points in response:

- I didn't dismiss the idea, I thought it was unlikely that it is something the group would do. 
- Even as co-chair, we work on a consensus process so if the group consensus were to create captioning standards, we might be able to do that (in a new charter period).
- Variables that would be important in that decision would be:
- - What is the current quality of the available standards.
- - How applicable are they across the world? (I'm not sure how they would work for internationalization, but it is a question that needs to be answered first.)
- Questioning the testability of a requirement in no way undermines the importance. Testable does not correlate with important, unfortunately. We have many cognitive-related barriers that are important but very difficult to test.
- If such a requirement were at the base level (like in WCAG 2), Netflix could be sued because one caption on one video goes over the face of a character. That's the nature of WCAG 2 style binary true/false statements, which (IMHO) is not how a captioning standard could or should work.
- We will indeed have to look at similar standards in other areas, such as audio-description. We have over 180 requirements already, many of which cover large areas (like captions), there is no way one group can create standards for all of them.

A key point to understand is that WCAG 2 became popular and well-used because it provides a set of pass/fail statements. However, WCAG 3 is being created (in part) because we know that approach cannot cover all the important aspects as well as we would like. Whether captions are available is a true/false thing, whether they are any good is much harder to fit into a binary form. That is the same for a lot of the requirements we work on, across all disabilities.

**Comment by JLS2024** [comment]
Statement:
"I didn't dismiss the idea, I thought it was unlikely that it is something the group would do."

Response:
It seems that by stating it was unlikely, the idea may have been unintentionally dismissed. Addressing this would require further consideration of its potential value to the group.

Statement:
"Even as co-chair, we work on a consensus process, so if the group consensus were to create captioning standards, we might be able to do that (in a new charter period)."

Response:
For consensus to be meaningful, it is crucial that the group includes representation from people who are deaf and hard of hearing. Without this, the decision-making process will likely not fully address their needs or challenges.

Statement:
"Variables that would be important in that decision would be: -- What is the current quality of the available standards. -- How applicable are they across the world? (I'm not sure how they would work for internationalization, but it is a question that needs to be answered first.)"

Response:
While the current quality of available standards is important, the purpose here is to create baseline standards that can evolve over time as needed. These standards would serve as a reference point globally, recognizing that different countries may be at different stages of implementation. The goal is to establish achievable standards for all.

Statement:
"Questioning the testability of a requirement in no way undermines the importance. Testable does not correlate with important, unfortunately. We have many cognitive-related barriers that are important but very difficult to test."

Response:
I agree that testability and importance don’t always align. In the case of captioning, the challenge lies in ensuring there is a clear pathway to test for accessibility, even if it’s not straightforward. While testability may be difficult, it should not deter us from prioritizing these essential accessibility measures.

Statement:
"If such a requirement were at the base level (like in WCAG 2), Netflix could be sued because one caption on one video goes over the face of a character. That's the nature of WCAG 2 style binary true/false statements, which (IMHO) is not how a captioning standard could or should work."

Response:
The Netflix lawsuit by the National Association of the Deaf (NAD) highlights the importance of having strong captioning standards. While no single organization should be seen as the gold standard, it is clear that robust guidelines help ensure consistent and meaningful accessibility.

Statement:
"We will indeed have to look at similar standards in other areas, such as audio-description. We have over 180 requirements already, many of which cover large areas (like captions), there is no way one group can create standards for all of them."

Response:
The group has done extensive work on visual standards, which is commendable. However, ensuring a balanced approach across various disabilities is essential. This includes dedicating sufficient attention to hearing-related needs, such as captioning.

Statement:
"A key point to understand is that WCAG 2 became popular and well-used because it provides a set of pass/fail statements. However, WCAG 3 is being created (in part) because we know that approach cannot cover all the important aspects as well as we would like. Whether captions are available is a true/false thing, whether they are any good is much harder to fit into a binary form. That is the same for a lot of the requirements we work on, across all disabilities."

Response:
Standards such as the ANA (American National Standards Institute) already use pass/fail measures, and this method works effectively in many cases. Ensuring that captions meet a certain standard, while imperfect, is achievable. What remains unclear from this discussion is whether there are members of the group who are deaf or hard of hearing. Their insights would greatly enhance the process and ensure that we avoid creating an echo chamber.
